examples/with-glamorous/pages/index.js

```diff
@@ -1,7 +1,14 @@
 import React from 'react'
-import { css } from 'glamor'
+import { style, rehydrate, css } from 'glamor'
 import glamorous from 'glamorous'
 
+// Adds server generated styles to glamor cache.
```

Comment:
@ads1018 You can DRY this up by including in a component (like a layout component) and using that on all the pages. See here: https://github.com/kentcdodds/glamorous-website/pull/14